api/routes/productRouter.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const productController = require('../controllers/productController');
+
+const router = express.Router();
+
+router.get('/all', productController.getAllProducts)
```

Comment:
전체 상품 리스트를 조회할 때 `/all`이라는 엔드포인트로 직접 명시하지 않아도 될 것 같아 보입니다!
```suggestion
router.get('/', productController.getAllProducts)
```